charactersheet/charactersheet/templates/dm/encounter_sections/environment_section.tmpl.html

```diff
@@ -0,0 +1,73 @@
+<div class="container-fluid">
+  <div class="h3">
+    Environment<br />
+    <small>Where is this encounter, and what's it like there?</small>
+  </div>
+  <!-- Begin Tabs -->
+  <ul class="nav nav-tabs tabs">
+    <li role="presentation" data-bind="click: selectPreviewTab, css: previewTabStatus">
+      <a href="#" aria-controls="itemsModalPreview" role="tab" data-toggle="tab">
+          <b>Preview</b>
+      </a>
+    </li>
+    <li role="presentation" data-bind="click: selectEditTab, css: editTabStatus">
+      <a href="#" aria-controls="itemsModalEdit" role="tab" data-toggle="tab">
+          <b>Edit</b>
+      </a>
+    </li>
+  </ul>
+  <div class="tab-content">
+    <div role="tabpanel" data-bind="css: previewTabStatus" class="tab-pane">
+      <div class="row row-padded">
+        <div data-bind="col-xs-12">
+          <div class="row row-padded">
+            <div class="col-xs-12"><b>Terrain:</b>&nbsp;<span data-bind="text: terrain"></span></div>
+          </div>
+          <div class="row">
+            <div class="col-xs-12"><b>Weather:</b>&nbsp;<span data-bind="text: weather"></span></div>
+          </div>
+          <hr />
+          <div>
+            <!-- ko if: imageUrl -->
+            <img data-bind="attr: { src: imageUrl, width: imageWidth }, css: imageClass"
+              class="img-responsive img-rounded">
+            <!-- /ko -->
+            <span data-bind="markdownPreview: description"
+               class="preview-modal-overflow-sm">
+            </span>
+          </div>
+        </div>
+      </div>
+    </div> <!-- End Preview Tab -->
+    <div role="tabpanel" data-bind="css: editTabStatus" class="tab-pane">
+      <form class="form-horizontal">
+        <div class="form-group">
+          <label class="col-sm-2 col-padded control-label">Image</label>
```

Comment:
Can we somehow indicate that this needs a URL? It's not clear as is.
Some small text under it might suffice. Up to you.

![image](https://cloud.githubusercontent.com/assets/7286387/20916328/0bc6e01e-bb40-11e6-9ab9-62492a1afe55.png)